<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>35</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>115</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Alias</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>About game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add words</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>370</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Team1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Round Points</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>34</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>34</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rounds  -</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Score - </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rounds  -</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Score - </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>34</pointsize>
       </font>
      </property>
      <property name="text">
       <string> Team2</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>player1</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>player2</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>player1</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>player2</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>34</pointsize>
       </font>
      </property>
      <property name="text">
       <string>   Team1</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>380</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>34</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>34</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Round seconds</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Final score</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>100</height>
      </size>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>40</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Add new words</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>241</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>115</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Alias</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Marker Felt</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>return to main menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>391</width>
        <height>301</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Alias is a fun word explanation game that is played 
in teams of 2 or more people. 
The aim is to make your teammates guess the 
word you are explaining by giving them hints 
and tips. For each correct guess the team moves
 forward on the game board, and the team that 
 reaches the finish space first wins the game!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Optima</family>
        <pointsize>60</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>GAME OVER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>35</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Win</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TeamName</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
